precompile/contracts/bibliophile/redstone.go

```diff
@@ -0,0 +1,25 @@
+package bibliophile
+
+import (
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/precompile/contract"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+var (
+	RED_STONE_VALUES_MAPPING_STORAGE_LOCATION  = common.HexToHash("0x4dd0c77efa6f6d590c97573d8c70b714546e7311202ff7c11c484cc841d91bfc") // keccak256("RedStone.oracleValuesMapping");
```

Comment:
are the `keccak256("method")` and `hash(slot_id)` both valid ways of reading values? How do we decide which method to use?

Reply:
That's a good observation. Normally, we "can't use" keccak256 of the method - in this case we are able to do so because red stone contracts are explicitly written in that manner. See [here](https://github.com/redstone-finance/redstone-oracles-monorepo/blob/fee81a2ec06a4c00b107583b41ff2e1ee42743ee/packages/on-chain-relayer/contracts/price-feeds/with-rounds/PriceFeedsAdapterWithRounds.sol#L8).
They explicitly write at a pre-determined storage location in the contract. Could very well have been `keccak256("harry potter obama sonic inu")`

Reply:
😮